### PR TITLE
ModalSpinner: better style

### PR DESCRIPTION
**Pull request by apepper**
Motivation: Less js payload due to no modal from react-boostrap.

### CODE REVIEW

File: src/Components/ModalSpinner.tsx [hunk 1]
```diff
@@ -1,22 +1,14 @@
...
-      <Modal
-        backdrop="static"
-        centered
-        dialogAs={() => <div className="loader" />}
-        fullscreen
-        keyboard={false}
-        show
-      />
-    </>
+    <dialog className="loader-dailog" ref={dialogRef}>
```
Comment (dcsaszar) [inline src/Components/ModalSpinner.tsx:9]: * [ ] Now that `Modal` no longer handles the keyboard input, we must prevent closing the dialog with Escape.

**Review by dcsaszar — CHANGES_REQUESTED** [review]
Nice. See comments for some small todos.

File: src/assets/stylesheets/theme/loader.scss [hunk 2]
```diff
@@ -87,3 +87,16 @@
...
+
+.loader-dailog,
+.loader-dailog:focus-visible {
+  background: transparent;
+  outline: none;
+  border: none !important;
+  width: 100%;
+  height: 100%;
+}
+.modal-backdrop {
```
[the remark this answers is not in the document]
Comment (agessler) [reply src/assets/stylesheets/theme/loader.scss:99]: Yes..
First... it didn't work in test. 
Second... I used just the bootstrap markup/style here. So nobody will be confused.

[the remark this answers is not in the document]
Comment (dcsaszar) [reply src/assets/stylesheets/theme/loader.scss:99]: It looks like this in Chrome:
image

[the remark this answers is not in the document]
Comment (agessler) [reply src/assets/stylesheets/theme/loader.scss:99]: Done now... :backdrop is not in editor visible like.. :after or :before.

**Comment by apepper** [comment]
[ac9e284](https://github.com/Scrivito/scrivito-portal-app/pull/510/commits/ac9e2845107ad3b6407eb9549c48b6c632300ca1)

Fixed Safari as well.

**Comment by apepper** [comment]
REVIEW ([848d8c5](https://github.com/Scrivito/scrivito-portal-app/pull/510/commits/848d8c5de7383c7d99ea6f9f4900dd7323e61ce5))

**Review by dcsaszar — APPROVED** [review]

**Pull request merged**